dotnet/src/Connectors/Connectors.Amazon/Bedrock/Core/Clients/BedrockChatCompletionClient.cs

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) Microsoft. All rights reserved.
+using Amazon.BedrockRuntime;
+using Amazon.BedrockRuntime.Model;
+using Connectors.Amazon.Core.Requests;
+using Connectors.Amazon.Core.Responses;
+using Connectors.Amazon.Models;
+using Connectors.Amazon.Models.AI21;
+using Connectors.Amazon.Models.Amazon;
+using Connectors.Amazon.Models.Anthropic;
+using Connectors.Amazon.Models.Cohere;
+using Connectors.Amazon.Models.Meta;
+using Connectors.Amazon.Models.Mistral;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Diagnostics;
+
+namespace Microsoft.SemanticKernel.Connectors.Amazon.Core;
+/// <summary>
```

Comment:
I think standard formatting is for there to be space between `namespace` and `/// <summary>`. I'm not 100% sure, but it made me think that we should look at how the main repo is applying formatting and hopefully we can auto-format this PR code the same way. You should be able to call `dotnet format` as described here: https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md#development-scripts